megatron/core/optimizer/cpu_offloading/hybrid_optimizer.py

```diff
@@ -90,7 +111,7 @@ def _split_parameters_updated_on_the_cpu_and_gpu(self):
         for param in self.params:
             if offloaded_params_numel < offload_threshold:
                 assert param.is_cuda
-                param_cpu_copy = param.detach().cpu()
+                param_cpu_copy = param.detach().cpu().pin_memory()
```

Comment:
If I remember correctly, there is a size limit for pin memory. Have we ever encountered a situation where pin memory does not work because it is too large?